x-pack/plugins/siem/public/components/stat_items/areachart.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { pure } from 'recompose';
+import { EuiSeriesChart, EuiAreaSeries } from '@elastic/eui/lib/experimental';
+import { AreaChartData, WrappedByAutoSizer, ChartOverlay } from '.';
+import { AutoSizer } from '../auto_sizer';
+
+const ChartBaseComponent = pure<{
+  data: AreaChartData[];
+  width: number | undefined;
+  height: number | undefined;
+}>(({ data, ...chartConfigs }) =>
+  chartConfigs.width && chartConfigs.height ? (
+    <EuiSeriesChart showDefaultAxis={false} {...chartConfigs}>
```

Comment:
This component is being deprecated fairly soon (we listed this in the docs in January of this year and targeted a June deprecation https://github.com/elastic/eui/issues/1469). You should contact @markov00 for an introduction to Elastic Charts.

Reply:
We just had that meeting yesterday :). The plan is to merge it like this but switch to elastic-charts as soon as it's available into Kibana as a dependency (maybe it already is since Code got merged?).